src/services/SyncManager.ts

```
@@ -1,52 +1,186 @@
+    const localOnlyItems: Local[] = [];
+    const remoteOnlyItems: Local[] = [];
+    const modifiedItems: { local: Local; remote: Local; isLocalNewer: boolean }[] = [];
+
+    // Check local items first
+    for (const localItem of localItems) {
+      const remoteItem = remoteMap.get(localItem.id);
+      if (!remoteItem) {
+        localOnlyItems.push(localItem);
+      } else {
+        // Check if local item is modified by comparing content, not just timestamps
+        const { [this.cfg.updatedAtColumn || 'updatedAt']: localUpdatedAt, ...localWithoutTs } = localItem as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)